templates/scripts/ci_releases/setup_repo_for_writing.sh

```diff
@@ -20,6 +20,14 @@ if [[ ( -z $GITLAB_PROJECT_RW_AND_API_TOKEN ) ]]; then
   exit 1
 fi
 
+if [[ ! -z "${GITLAB_PROJECT_RW_AND_API_TOKEN}" ]]; then
```

Comment:
this check is not necessary because at line 15 we check for the opposite to bail out.

Reply:
thanks for catching, fixed